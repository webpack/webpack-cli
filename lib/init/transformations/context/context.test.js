"use strict";

const defineTest = require("../../../utils/defineTest");

defineTest(
	__dirname,
	"context",
	"context-0",
	"path.resolve(__dirname, 'app')",
	"init"
);
defineTest(__dirname, "context", "context-1", "'./some/fake/path'", "init");
defineTest(__dirname, "context", "context-2", "contextVariable", "init");
