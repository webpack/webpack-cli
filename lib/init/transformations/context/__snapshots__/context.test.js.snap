// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`context transforms correctly using "context-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    context: path.resolve(__dirname, 'app')
}
"
`;

exports[`context transforms correctly using "context-1" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    context: './some/fake/path'
}
"
`;

exports[`context transforms correctly using "context-2" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    context: contextVariable
}
"
`;
